icd/api/include/binary_cache_serialization.h

```diff
@@ -56,10 +56,10 @@
 */
 #pragma once
 
-#include "include/khronos/vulkan.h"
 #include "include/vk_defines.h"
 #include "palMetroHash.h"
 #include "palUtil.h"
+#include "vulkan.h"
```

Comment:
Why are you changing how we include vulkan.h?  Within the icd we always use include/khronos/vulkan.h.

Reply:
I need to expose `api/include/khronos` as an include path to the xgl_cache_support library, so that cache_creator can include `vulkan.h` from within the vkgcDefs header. Since binary_cache_serialization is in this lib, it can now also include vulkan using the same path. I can revert this if it violates the coding style rule you mentioned.

Reply:
How this is done pre-dates me and the folks who started did the initial vulkan driver work aren't here anymore.  I'm hesitant to change this if it isn't necessary (and it seems it isn't).

Reply:
I undid this change.